app/db/models.py

```diff
@@ -31,28 +30,55 @@ class JobStatus(enum.Enum):
     FAILED = "FAILED"
 
 
-class AOI(Base):
-    __tablename__ = "aois"
+class ModelType(enum.Enum):
+    SEGMENTATION = "SEGMENTATION"
+    CLASSIFICATION = "CLASSIFICATION"
+
+
+class Satellite(Base):
+    __tablename__ = "satellites"
 
     id = Column(Integer, primary_key=True)
-    name = Column(CONSTRAINT_STR, nullable=False)
-    created_at = Column(DateTime, nullable=False, default=datetime.datetime.now)
-    is_deleted = Column(Boolean, nullable=False, default=False)
-    geometry = Column(Geometry(geometry_type="POLYGON", srid=4326), nullable=False)
+    name = Column(String, nullable=False, unique=True)
 
-    jobs = relationship("Job", backref="aoi", cascade="all, delete, delete-orphan")
+    bands = relationship(
+        "Band", backref="satellite", cascade="all, delete, delete-orphan"
+    )
+    images = relationship(
+        "Image", backref="satellite", cascade="all, delete, delete-orphan"
+    )
+
+    def __init__(self, name: str):
+        self.name = name
+
+
+class Band(Base):
```

Comment:
I really like that we are now also saving information about Satellites and bands in the DB. Just so I understand it a bit better, at least for the satellite we do this just because we can right? We not necessarily need a relation to the satellite right? 

Btw.. after this is is merged this might be interesting for the user as well and we could display this  as metadata in the sidebar.

Reply:
We do need the Satellites table if we want to be able to let a Model predict on different Satellite images in the future. With the current setup this means a new Model instance should be created with a relation to the expected bands > satellite